sdk/tests/js/webgl-test-utils.js

```diff
@@ -40,10 +40,11 @@ var loggingOff = function() {
  * @return {string} The enum as a string.
  */
 var glEnumToString = function(gl, value) {
-  // Avoid returning "NO_ERROR" if the arguments are totally wrong.
-  if (gl.NO_ERROR === undefined || value === undefined) {
-    return undefined;
-  }
+  if (gl.NO_ERROR === undefined)
+    throw new Error('glEnumToString: `gl` must be a WebGL2?RenderingContext');
+  if (value === undefined)
+    throw new Error('glEnumToString: `value` must not be undefined');
+
```

Comment:
There seems to be a problem here: `value` could be an enum defined by extensions, and it's an exception here if `glEnumToString()` is called like [this](https://github.com/KhronosGroup/WebGL/blob/a782558f12df78e43d23089a70cd975e257c2381/sdk/tests/js/tests/webgl-compressed-texture-size-limit.js#L160) (note the second `glEnumToString()` of the line).